tests/qunit/tests/js/api.util.js

```diff
@@ -173,6 +173,10 @@ require(['gh.core', 'gh.api.tests'], function(gh, testAPI) {
 
         // Verify that a valid week number is returned when specifying an out-of-term date that leans close to the start of a term
         weekNumber = gh.utils.getAcademicWeekNumber(gh.utils.convertISODatetoUnixDate('2015-01-12T10:30:00.000Z'));
+        assert.strictEqual(weekNumber, 1, 'Verify that a valid week number is returned when specifying an out-of-term date');
+
+        // Verify that a valid week number is returned when specifying an out-of-term date that leans close to the start of a term, using precise
+        weekNumber = gh.utils.getAcademicWeekNumber(gh.utils.convertISODatetoUnixDate('2015-01-12T10:30:00.000Z'), true);
         assert.strictEqual(weekNumber, 0, 'Verify that a valid week number is returned when specifying an out-of-term date');
```

Comment:
Can you add to the description of the assert that it's testing the `precise` parameter?